HACKING.md

```diff
@@ -0,0 +1,45 @@
+
+# Working on charmhub.io
+
+## Use staging APIs
+
+To use staging APIs locally you can add the following lines to an `.env.local` file:
+
+```bash
+CHARMSTORE_API_URL=https://api.staging.snapcraft.io/
+CHARMSTORE_PUBLISHER_API_URL=https://api.staging.charmhub.io/
+CANDID_API_URL=https://api.staging.jujucharms.com/identity/
+```
+
+## Using Sentry error tracker
+
+For development purposes, visit https://sentry.io/signup/, signup and setup a project. By then you will have a sentry DSN string like:
+
+```
+https://<user>:<secret>@sentry.io/<project_id>
+```
+
+Create or update you `.env.local` file:
```

Comment:
'your' not 'you'

Reply:
thanks, will update it in a separate pr as this is merged already